test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/CSharp7.cshtml

```diff
@@ -0,0 +1,32 @@
+@{
+    var nameLookup = new Dictionary<string, (string FirstName, string LastName, object Extra)>()
+    {
+        ["John Doe"] = ("John", "Doe", true)
+    };
+
+    int Sixteen = 0b0001_0000;
+    long BillionsAndBillions = 100_000_000_000;
+    double AvogadroConstant = 6.022_140_857_747_474e23;
+    decimal GoldenRatio = 1.618_033_988_749_894_848_204_586_834_365_638_117_720_309_179M;
+}
+
+@if (nameLookup.TryGetValue("John Doe", out var entry))
```

Comment:
Can we add some Html and some comments to the mix? Would be nice to see if that doesn't break things.